### DISCUSSION TITLE
#include <bits/stdc++.h> not working vs code c++

### DISCUSSION THREAD

**Discussion by visudhi** [microsoft/vscode-cpptools#11279] [Q&A] [answered]
![Uploading 1.png…]()
Hi All,

I'm trying to use **list** in vscode , im trying to import the header #include <bits/stdc++.h> seems to be giving an error.

Can anyone help how to mitigate this issue?

**Comment by sean-mcmanus** [comment]
What is your code? What is the error?

**Comment by visudhi** [comment, marked as answer]
Hi ,

Now its resolved, just created an include folder and a bits folder . 

Within the bits craeted a file stdc++  with all the extensions  inside it.

I Used this link
https://gist.github.com/Einstrasse/ac0fe7d7450621a39364ed3b05cacd11


Music\C++\dsa_cpp\ math\include\bits  (just placed the folder where im running my code)
